src/main/java/org/jitsi/jigasi/JigasiBundleActivator.java

```diff
@@ -104,7 +111,7 @@ public void stop(BundleContext bundleContext)
     {
         logger.info("Stopping JigasiBundleActivator");
 
-        gateway.stop();
+        sipGateway.stop();
```

Comment:
I know that TranscriptionGateway#stop() doesn't do anything right now, but I think we should call it here anyway.